README.md

```diff
@@ -1298,17 +1301,17 @@ A diagram with all the terms is likely worth it.
 
 ### Early exercise
 
-🄳 Sometimes, to help you lower your tax burden, the company makes it possible to **early
-exercise (or forward exercise)** stock options.
-This means you exercise them even before they vest, which means you become a stockholder
-earlier than you would otherwise.
+🄳 Sometimes, to help employees lower their tax burden, the company makes it possible to
+**early exercise (or forward exercise)** stock options.
+This means the options can be exercised even before they vest, bringing up the date at
```

Comment:
Fixed.

Reply:
Actually looks like this one was missed.

Employees -> people who work for the company (or similar)

"bringing up" can also be misread. (Bringing up with whom?)

How about:

Sometimes, to help reduce the tax burden of stock options, a company makes it possible to **early exercise (or forward exercise)** options, which means the option holder can exercise even before they vest. The result is to become a stockholder sooner, with the vesting then applying to actual stock rather than options.

Reply:
Whoops, missed the first one! Fixed, but double check the final sentence... You mean the vesting schedule applies to the stock, meaning, controls when it can be sold? That's new content.